src/lib/connectors/repos/licence-version-purpose-conditions.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+const { bookshelf } = require('../bookshelf');
+const queries = require('./queries/licence-version-purpose-conditions');
+const helpers = require('./lib/helpers');
+
+const LicenceVersionPurposeConditions = require('../bookshelf/LicenceVersionPurposeConditions');
+/**
+ * Gets a specific licence version purpose condition by its ID.
+ * @param {String} id
+ * @return {Promise<Array>}
+ */
+const findOneById = async id => helpers.findOne(LicenceVersionPurposeConditions, 'licenceVersionPurposeConditionId', id);
+
+const findManyByLicenceId = (licenceId, code = null) => {
+  return !code
+    ? bookshelf.knex.raw(queries.findLicenceVersionPurposeConditionsByLicenceId, { licenceId })
```

Comment:
In the queries below, columns are renamed to make them camel-cased.

If it makes life easier there is a helper with methods `raw.singleRow` `raw.multiRow` which can run these queries and return camelCased column names

Reply:
Makes sense, thank you! I will tweak this as suggested 👍🏽